test/controllers/overseas.entity.due.diligence.controller.spec.ts

```diff
@@ -88,7 +92,12 @@ describe("OVERSEAS_ENTITY_DUE_DILIGENCE controller", () => {
   describe("POST tests", () => {
 
     test(`redirect to ${ENTITY_PAGE} page after a successful post from ${OVERSEAS_ENTITY_DUE_DILIGENCE_PAGE} page`, async () => {
-      mockPrepareData.mockReturnValueOnce( OVERSEAS_ENTITY_DUE_DILIGENCE_OBJECT_MOCK );
+      const dueDiligenceMock = OVERSEAS_ENTITY_DUE_DILIGENCE_REQ_BODY_OBJECT_MOCK;
```

Comment:
I'd make a copy of the mock as when you modify it, that data will be picked up by other tests.
use 
const dueDiligenceMock = { ...OVERSEAS_ENTITY_DUE_DILIGENCE_REQ_BODY_OBJECT_MOCK }
so that the test modifies this new object instead of the real mock